homeassistant/components/matter/binary_sensor.py

```diff
@@ -95,6 +95,17 @@ def _update_from_device(self) -> None:
         entity_class=MatterBinarySensor,
         required_attributes=(clusters.OccupancySensing.Attributes.Occupancy,),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.BINARY_SENSOR,
+        entity_description=MatterBinarySensorEntityDescription(
+            key="WaterLeakSensor",
+            device_class=BinarySensorDeviceClass.MOISTURE,
+            # The first bit = if occupied
+            measurement_to_ha=lambda x: (x & 1 == 1) if x is not None else None,
```

Comment:
This is not the case for BooleanState, also we don't need to invert as we have to in Occupancy sensing. So I think you can just not provide a lambda to `measurement_to_ha`.

```suggestion
```

Reply:
Thanks for the code review.